packages/core-cairo/src/account.ts

```diff
@@ -0,0 +1,186 @@
+import { Contract, ContractBuilder } from './contract';
+import { CommonOptions, withCommonDefaults } from './common-options';
+import { defaults as commonDefaults } from './common-options';
+import { setAccountUpgradeable } from './set-upgradeable';
+import { setInfo } from './set-info';
+import { defineComponents } from './utils/define-components';
+import { printContract } from './print';
+import { addSRC5Component } from './common-components';
+
+
+export const accountOptions = ['stark', 'eth'] as const;
+export type Account = typeof accountOptions[number];
```

Comment:
This has a similar name as the AccountOptions interface below, so would be clearer to rename it.
```suggestion
export const accountTypes = ['stark', 'eth'] as const;
export type Account = typeof accountTypes[number];
```